pkg/client/clientset/versioned/cluster/typed/meta/v1alpha1/apiresource.go

```diff
@@ -0,0 +1,72 @@
+//go:build !ignore_autogenerated
```

Comment:
Please double-check your rebase, this file was deleted in #988 and should not be coming back.

Reply:
fixed